src/components/ActionCard.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { colors, ColorOption } from '@site/src/constants/colors';
+import { Icon } from './Icon';
 
 interface ActionCardProps {
+  icon?: string;
```

Comment:
This is similar to the optional title thing from the other PR. If you make it optional, can you add a conditional for Icon component/section in the UI.